client/src/api/block_builder/input_selection/core/error.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Error handling in iota-client crate.
+
+use std::fmt::Debug;
+
+use iota_types::block::output::{ChainId, OutputId, TokenId};
+use primitive_types::U256;
+use serde::Serialize;
+
+use crate::{api::input_selection::Requirement, error::display_string};
+
+/// Error type of the iota client crate.
```

Comment:
Comments should probably mention ISA

Reply:
Indeed

Reply:
And the `//! Error handling in iota-client crate.`?

Reply:
Oh ffs

Reply:
Done